src/includes/set-context/java.mdx

```diff
@@ -0,0 +1,9 @@
+```java
+import io.sentry.Sentry;
+
+Sentry.configureScope(scope -> {
+    scope.setContexts("name", "Mighty Fighter");
```

Comment:
This won't work I'm afraid. A context takes an object, just a string will get Sentry to yell

Reply:
it does, I guess, you'd just need to cast it when you do a get for this key

Reply:
I checked, it works on the SDK level but indeed shown in Sentry as an error. Wondering how is it in other SDKs as this piece of code is a direct translation from Python or dotnet sample.